qCC/ccCommandLineCommands.cpp

```diff
@@ -3682,6 +3684,66 @@ CommandC2CDist::CommandC2CDist()
 	: CommandDist(false, QObject::tr("C2C distance"), COMMAND_C2C_DIST)
 {}
 
+CommandCPS::CommandCPS()
+    : ccCommandLineInterface::Command(QObject::tr("Closest Point Set"), COMMAND_CLOSEST_POINT_SET)
+{}
+
+bool CommandCPS::process(ccCommandLineInterface &cmd)
+{
+    cmd.print(QObject::tr("[CLOSEST POINT SET]"));
+
+    // COMPARED CLOUD / REFERENCE ENTITY
+    ccPointCloud* compPointCloud = nullptr;
+    ccPointCloud* refPointCloud = nullptr;
+    CLCloudDesc compDesc;
+    CLCloudDesc refDesc;
+
+    if (cmd.clouds().size() < 2)
```

Comment:
you can move these 2 tests above (after the first 'print' command) before declaring the  local variables